tests/parm/ice_in.IN

```diff
@@ -1,48 +1,61 @@
 &setup_nml
-    days_per_year  = 365
-    use_leap_years = .true.
-    year_init      = @[SYEAR]
-    month_init     = @[SMONTH]
-    day_init       = @[SDAY]
-    sec_init       = @[SECS]
-    dt             = @[DT_CICE]
-    npt            = @[CICE_NPT]
-    ndtd           = 1
-    runtype        = '@[CICE_RUNTYPE]'
-    runid          = '@[CICE_RUNID]'
-    ice_ic         = 'cice_model.res.nc'
-    restart        = .true.
-    restart_ext    = .false.
-    use_restart_time = @[CICE_USE_RESTART_TIME]
-    restart_format = 'nc'
-    lcdf64         = .false.
-    numin          = 21
-    numax          = 89
-    restart_dir    = '@[CICE_RESTART_DIR]'
-    restart_file   = '@[CICE_RESTART_FILE]'
-    pointer_file   = './ice.restart_file'
-    dumpfreq       = '@[CICE_DUMPFREQ]'
-    dumpfreq_n     = @[CICE_DUMPFREQ_N]
-    dump_last      = .false.
-    bfbflag        = 'off'
-    diagfreq       = @[CICE_DIAGFREQ]
-    diag_type      = 'file'
-    diag_file      = 'ice_diag.d'
-    print_global   = .true.
-    print_points   = .true.
-    latpnt(1)      =  90.
-    lonpnt(1)      =  0.
-    latpnt(2)      = -65.
-    lonpnt(2)      = -45.
-    histfreq       = 'm','d','h','x','x'
-    histfreq_n     = @[CICE_HISTFREQ_N]
-    hist_avg       = @[CICE_HIST_AVG]
-    history_dir    = '@[CICE_HISTORY_DIR]'
-    history_file   = 'iceh'
-    write_ic       = .true.
-    incond_dir     = '@[CICE_INCOND_DIR]'
-    incond_file    = 'iceh_ic'
-    version_name   = 'CICE_6.0.2'
+    days_per_year	= 365
+    use_leap_years	= .true.
```

Comment:
These all have different spacing where the = sign is. At least on my phone. Can you double check?

Reply:
I aligned the namelist on the ``=`` sign.

Reply:
![Screenshot from 2024-04-05 08-04-32](https://github.com/ufs-community/ufs-weather-model/assets/64433609/5d52ee1d-0b6b-441c-ac1a-0ab0957e9b46)
This is how it looks on my laptop. Might want to verify your editor has the = sign aligned for the entire section.

Reply:
Tabs vs spaces.

Reply:
@DusanJovic-NOAA Thanks. @BrianCurtis-NOAA Does that fix it?